### DISCUSSION TITLE
Delly v1.0.3 vs v0.9.1

### DISCUSSION THREAD

**Discussion by Faizal-Eeman** [uclahs-cds/pipeline-call-sSV#39] [Show and tell]
# Description
This discussion compares Delly v0.9.1 to v1.0.3. No difference is observed in significant variant calls i.e. having good mapping quality, but runtime has increased with v1.0.3. The larger, A-full datasets appear to be running with a relatively shorter runtime with F16 node configurations as compared to running on F32/72 node configurations.

### Variant Calls
**1. SVs with Good Mapping Quality -** No differences in the SV calls were found between v0.9.1 and v1.0.3 for those variants with Mapping Quality (MAPQ) > 0

- `/hot/software/pipeline/pipeline-call-sSV/Nextflow/development/unreleased/mmoootor-upgrade-delly-0.9.1-to-1.0.3/delly-0.9.1-vs-1.0.3_validation/A-mini_S2.T-0/delly0.9.1-vs-1.0.3_MAPQ_gt_zero_filtered.sha512`
- `/hot/software/pipeline/pipeline-call-sSV/Nextflow/development/unreleased/mmoootor-upgrade-delly-0.9.1-to-1.0.3/delly-0.9.1-vs-1.0.3_validation/A-full_T5.T/delly0.9.1-vs-1.0.3_MAPQ_gt_zero_filtered.sha512`

**2. SVs with Bad Mapping Quality -** Differences were seen in the number of SVs called by v0.9.1 and v1.0.3 for only those variants with Mapping Quality (MAPQ) = 0. The difference in the number of variants called, appears to be predominantly due to the difference in the Quality Scores of these variants between v0.9.1 and v1.0.3.

- `/hot/software/pipeline/pipeline-call-sSV/Nextflow/development/unreleased/mmoootor-upgrade-delly-0.9.1-to-1.0.3/delly-0.9.1-vs-1.0.3_validation/A-mini_S2.T-0/delly0.9.1-vs-1.0.3_difference_only_in_MAPQ_zero.txt`
- `/hot/software/pipeline/pipeline-call-sSV/Nextflow/development/unreleased/mmoootor-upgrade-delly-0.9.1-to-1.0.3/delly-0.9.1-vs-1.0.3_validation/A-full_T5.T/delly0.9.1-vs-1.0.3_difference_only_in_MAPQ_zero.txt`

### Runtime with **Stringent Filters** applied
**A-mini**
- Delly v0.9.1 (F2 node) - 13m 33.9s
- Delly v1.0.3 (F2 node) - 20m 32s

**A-full**
- Delly v1.0.3 (F16 node) - 17h 53m 49s
- Delly v0.9.1 (F72 node) - 18h 16m 23.4s
- Delly v1.0.3 (F72 node) - 18h 16m 15s
- Delly v1.0.3 (F32 node) - 20h 14m 1s

**Real Sample - ILHNLNEV000001-T001-P01-F**
- Delly v1.0.3 (F16 node) - 8h 46m 31s
- Delly v0.8.7 (F32 node) - 8h 37m 34s
- Delly v1.0.3 (F32 node) - 9h 38m 29s

## Stringent Filters
- map_qual = 20
- min_clique_size = 5
- mad_cutoff = 15

## Related PRs
#35 Update Delly 0.9.1 to 1.0.3
#36 Add F16 config

**Comment by tnyamaguchi** [comment]
@Faizal-Eeman Nice work! Can you add the filtering info (default vs stringent) as well ? We've changed a few thresholds to reduce the run time before (see https://github.com/uclahs-cds/pipeline-call-sSV/issues/10)

**Reply by tnyamaguchi** [reply]
Never mind. I missed it but the stringent filter was applied to the runs.

**Reply by Faizal-Eeman** [reply]
The above comparisons are all based on the same stringent filters - stringent vs stringent.
I've also updated the types of stringent filters in this discussion.
